### PR TITLE
Update REDMEA with pre-receive installation guide

### PR DESCRIPTION
**Pull request by MaorDavidzon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,6 +166,69 @@ repos:
...
+
+        ```sh
+        #!/bin/sh
+
+        # optional
+        # Update the server URL only if you have Cycode self managed
+        # CYCODE_API_URL = "<cycode server url>"
+        export CYCODE_CLIENT_ID="<client_id>"
+        export CYCODE_CLIENT_SECRET="<client_secret>"
+        export CYCODE_API_URL="https://api.cycode.com"
```
Comment (MichalBor) [inline README.md:195]: remove

File: README.md [hunk 2]
```diff
@@ -166,6 +166,69 @@ repos:
...
+        - `chown git pre-receive` (change file owner).
+        - `chgrp git pre-receive` (change file group owner).
+        - Verify it by running `ls -l`. The output should be `-rwxr-xr-x 1 git git 662 Mar 2 09:15 pre-receive` (first `git` is the file's owner, second `git` is the file's group owner).
+
+3. Configure the Cycode token.
+    - There are two methods to set it:
+        - In the script above, fill in the client ID and secret.
+        - Set environment variables on the instance.
+
+4. [Optional] Update Cycode API url (relevant only for on-prem customers) - Update `CYCODE_API_URL` in the pre-receive file.
```
Comment (MichalBor) [inline README.md:212]: remove?

File: README.md [hunk 3]
```diff
@@ -166,6 +166,69 @@ repos:
...
+
+#### Install pre-recive hook gloaly (For all the repositories)
```
Comment (MichalBor) [inline README.md:214]: globaly

File: README.md [hunk 4]
```diff
@@ -166,6 +166,69 @@ repos:
...
+
+1. Set the global hooks directory in the Git server instance.
+    - For GitLab Enterprise:
```
Comment (MichalBor) [inline README.md:217]: not exactly 😓 
add this link instead https://docs.gitlab.com/ee/administration/server_hooks.html

File: README.md [hunk 5]
```diff
@@ -166,6 +166,69 @@ repos:
...
+        - Set in `/etc/gitlab/gitlab.rb` the `gitaly['custom_hooks_dir']` value or just use the default location by uncommenting it.
+        - Run `gitlab-ctl reconfigure`.
+
+2. Add the pre-receive hook according to the Git server requirements.
+    - For GitLab Enterprise:
+        - Go to the directory and create a directory named `pre-receive.d`.
+        - Inside the directory, follow the steps of "Install for a specific repository" above.
```
Comment (MichalBor) [inline README.md:224]: follow the steps of "Create the pre-receive hook" under "Install for a specific repository"

**Review by MichalBor — CHANGES_REQUESTED** [review]
💪🏻

File: README.md (hunk 2 above)
Comment (MaorDavidzon) [reply README.md:212]: Why?

Comment (MichalBor) [reply README.md:212]: you write it as a comment in the script

File: README.md [hunk 6]
```diff
@@ -166,6 +166,69 @@ repos:
...
+
+        ```sh
+        #!/bin/sh
+
+        # optional
+        # Update the server URL only if you have Cycode self managed
+        # CYCODE_API_URL = "<cycode server url>"
+        export CYCODE_CLIENT_ID="<client_id>"
+        export CYCODE_CLIENT_SECRET="<client_secret>"
+        export CYCODE_API_URL="https://api.cycode.xyz"
```
[the remark this answers is not in the document]
Comment (MaorDavidzon) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (MaorDavidzon) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (MaorDavidzon) [reply README.md:211]: Still, I prefer to write it here also

File: README.md (hunk 3 above)
Comment (MaorDavidzon) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (MaorDavidzon) [reply README.md]: Updated.

File: README.md (hunk 5 above)
Comment (MaorDavidzon) [reply README.md:224]: done

File: README.md [hunk 7]
```diff
@@ -211,12 +210,13 @@ repos:
...
 
-#### Install pre-recive hook gloaly (For all the repositories)
+#### Install pre-recive hook globaly (For all the repositories)
 
 1. Set the global hooks directory in the Git server instance.
-    - For GitLab Enterprise:
+    - For GitLab Enterprise (https://docs.gitlab.com/ee/administration/server_hooks.html#create-global-server-hooks-for-all-repositories) :
         - Set in `/etc/gitlab/gitlab.rb` the `gitaly['custom_hooks_dir']` value or just use the default location by uncommenting it.
         - Run `gitlab-ctl reconfigure`.
+        - Follow the steps [here](#create-the-pre-receive-hook)
```
Comment (MichalBor) [inline README.md:219]: the hook works?

**Review by MichalBor — APPROVED** [review]

**Pull request merged**